lms/djangoapps/teams/tests/test_views.py

```diff
@@ -509,7 +516,7 @@ def test_expand_public_user(self):
         ('queryable', ['Search']),
         ('Tonga', ['Search']),
         ('Island', ['Search']),
-        ('search queryable', []),
+        ('not-a-query', []),
         ('team', ['Another Team', 'Public Profile Team']),
```

Comment:
I meant to comment on this PR, but you might have been making changes, so Github got confused and moved some things around. I did cover 'a learner leaves the team' already, but the two situations you called out were not. Thanks for taking care of that!